cvat-data/src/ts/cvat-data.ts

```diff
@@ -155,17 +155,43 @@ export class FrameDecoder {
         }
     }
 
+    private validateFrameNumbers(frameNumbers: number[]): void {
+        if (!frameNumbers || !frameNumbers.length) {
+            throw new Error('frameNumbers must not be empty');
+        }
+
+        // ensure is ordered
+        for (let i = 1; i < frameNumbers.length; ++i) {
+            const prev = frameNumbers[i - 1];
+            const current = frameNumbers[i];
+            if (current <= prev) {
+                throw new Error(
+                    'frameNumbers must be sorted in ascending order, ' +
+                    `got a (${prev}, ${current}) pair instead`,
+                );
+            }
+        }
+    }
+
+    private arraysEqual(a: number[], b: number[]): boolean {
+        return (
+            a.length === b.length &&
+            a.every((element, index) => element === b[index])
+        );
+    }
+
```

Comment:
Is there any reason to make it class method?

Reply:
I thought it can be optimized somehow for this case, but maybe we can just check `chunkNumber` instead, which was added later.

Reply:
Removed, probably not needed anymore.